packages/server/src/server.ts

```diff
@@ -202,10 +209,6 @@ const logAndHandleResponse = async (methodName: any, methodParams: any, methodFu
     }
     return response;
   } catch (e: any) {
-    ms = Date.now() - app.getStartTimestamp();
-    methodResponseHistogram.labels(methodName, responseInternalErrorCode).observe(ms);
```

Comment:
Removing these calls to getStartTimestamp no longer requires the getStartTimestamp method in the koaJsonRpc, so it can be removed as well.

Reply:
Removed, thanks